<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>注册界面</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="register_title">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{border:None}</string>
   </property>
   <property name="text">
    <string>用户注册</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>161</width>
     <height>58</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="register_ok">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancle">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>291</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QLineEdit" name="cp">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="maxLength">
       <number>12</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="register_user">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>用户名：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="userName">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="password">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="maxLength">
       <number>12</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="register_repassword">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>确认密码：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="register_password">
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>密码：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>userName</tabstop>
  <tabstop>password</tabstop>
  <tabstop>cp</tabstop>
  <tabstop>register_ok</tabstop>
  <tabstop>cancle</tabstop>
 </tabstops>
 <resources>
  <include location="../../../UI/ui/source/apprcc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>register_ok</sender>
   <signal>clicked()</signal>
   <receiver>Register</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>468</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>470</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancle</sender>
   <signal>clicked()</signal>
   <receiver>userName</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancle</sender>
   <signal>clicked()</signal>
   <receiver>password</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>475</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancle</sender>
   <signal>clicked()</signal>
   <receiver>cp</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>407</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
